projects/common/src/telemetry/providers/rudderstack/rudderstack-provider.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { Dictionary } from '../../../utilities/types/types';
+
+import { apiObject, identify, load, page, track } from 'rudder-sdk-js';
+import { TelemetryProviderConfig, UserTelemetryProvider, UserTraits } from '../../telemetry';
+
+export interface RudderStackConfig extends TelemetryProviderConfig {
+  writeKey: string;
+}
+
+@Injectable({ providedIn: 'root' })
+export class RudderStackTelemetry implements UserTelemetryProvider<RudderStackConfig> {
+  public initialize(config: RudderStackConfig): void {
+    try {
+      load(config.writeKey, config.orgId, { configUrl: config.orgId });
+    } catch (error) {
+      /**
+       * Fail silently
+       */
```

Comment:
@jaywalker21 Just a thought instead of failing silently can we capture it and send to sentry or something similar ?

Reply:
Currently, HyperTrace as a whole doesn't have sentry integration.
We can take up overall integration w/ Sentry later on.